food-truck-api/src/test/java/food/truck/api/endpoint/AuthenticationEndpointTest.java

```diff
@@ -0,0 +1,45 @@
+package food.truck.api.endpoint;
+
+import food.truck.api.security.AuthenticationFilter;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.lang.annotation.Inherited;
+
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.Assert.*;
+
+public class AuthenticationEndpointTest extends EndpointTest {
+    protected String token;
+
+    @Test
+    public void register() {
+        String path = base + "register";
+        var data = new AuthenticationEndpoint.RegistrationData("testUser", "test@example.com", "password");
+        ResponseEntity<String> response = template.postForEntity(path, data, String.class);
+        System.out.println(response);
+        assertThat(response.getBody(), startsWith("Created user"));
+    }
+
+    @Test
+    public void loginSuccess() {
+        register();
```

Comment:
It would probably be more proper to use the user service to directly add a user, so this test isn't dependent on the register test